canonical-web-and-design/ubuntu.com
# Move iot/robotics to /robotics and add redirect

## Done

Move /internet-of-things/robotics to /robotics and added redirect

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/robotics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that you are redirected to /robotics


## Issue / Card

Fixes #2229
